azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/configuration/LocalizationConfiguration.java

```diff
@@ -26,38 +28,36 @@
  * @see CallComposite
  */
 public final class LocalizationConfiguration {
-    private final LanguageCode languageCode;
     private int layoutDirection;
```

Comment:
Can we make this variable final as well?

Reply:
Because we don't need it in the every constructor so that we cannot make it final